l5-operator/bundle/manifests/l5-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,198 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "pets.bestie.com/v1",
+          "kind": "Bestie",
+          "metadata": {
+            "name": "bestie"
+          },
+          "spec": {
+            "agencyname": "Animal Humane Society",
```

Comment:
interesting is this meant to be a multi tenant thing where every "agency" will have its own instance ?

Reply:
Exactly!